src/secrets/CommandList.ts

```diff
@@ -70,9 +79,7 @@ class CommandList extends CommandPolykey {
           process.stdout.write(
             binUtils.outputFormatter({
               type: 'list',
-              data: data.map(
-                (secretsListMessage) => secretsListMessage.secretName,
-              ),
+              data: data.map((secrets) => secrets.path),
```

Comment:
You like your maps, why wasn't this logic done in a previous map over the data or when it was pushed into the array? You can consolidate it all together.

Reply:
Oh, this is a remnant from when I had the long formatting of the files, as I had to retain all the data for long mode, but I didn't update this logic after its removal.